index.js

```diff
@@ -3,39 +3,38 @@ const reviews = [
   {
     id: 1,
     name: 'Mbah denzel',
-    profession: `Lorem ipsum dolor sit amet consectetur adipisicing`,
+    profession: 'general manager at All cocoa business, Kumba',
     text: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
     img: 'images/denzel.jpg',
   },
   {
     id: 2,
-    name: 'Mbah denzel',
-    profession: `Lorem ipsum dolor sit amet consectetur adipisicing`,
+    name: 'Derick dice sk',
+    profession: 'CEO at Green Engineering and Consultancy, Buea',
     text: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
-    img: 'images/denzel.jpg',
+    img: 'images/derick.jpg',
   },
   {
     id: 3,
     name: 'katche elvis',
-    profession: `Lorem ipsum dolor sit amet consectetur adipisicing`,
+    profession: 'Director at Good Morning Zang Construction',
     text: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
-    img: 'images/denzel.jpg',
+    img: 'images/elvis.jpg',
   },
   {
     id: 4,
     name: 'louis katche',
-    profession: `Lorem ipsum dolor sit amet consectetur adipisicing`,
+    profession: 'Technical Advicer at lT and Services, Limbe',
     text: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
-    img: 'images/denzel.jpg',
+    img: 'images/louis.jpg',
   },
   {
     id: 5,
     name: 'katche festus',
-    profession: `Lorem ipsum dolor sit amet consectetur adipisicing`,
+    profession: 'Student at Emmacroy Technologies',
     text: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
-    img: 'images/denzel.jpg',
+    img: 'images/festus.jpg',
   },
```

Comment:
- [ ] Your reviewers' list is good, but kindly make sure you have 6 reviewers to display in a layout of 2 columns with 3 rows of articles on the desktop version like the template.